lib/angular/fieldLocation/fieldLocationController.js

```diff
@@ -15,19 +15,43 @@ function FieldLocationController($scope) {
   fieldLocationCtrl.setLocation = function(event) {
```

Comment:
Comment is done but a tiny bit innacurate since it doesn't `return` an error message but rather sets it on the model.